config/initializers/sorcery.rb

```diff
@@ -0,0 +1,20 @@
+Rails.application.config.sorcery.submodules = [:external]
+Rails.application.config.sorcery.configure do |config|
+config.external_providers = [:facebook, :google]
+
+config.facebook.key = Rails.application.secrets.facebook_key
+config.facebook.secret = Rails.application.secrets.facebook_secret
+config.facebook.callback_url = "http://0.0.0.0:3000/oauth/callback?provider=facebook"
+config.facebook.user_info_mapping = {:email => "name"}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.